test_scripts/API/Expanded_proprietary_data_exchange/010_OnAppRegistered_without_requestSubType.lua

```diff
@@ -0,0 +1,39 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0037-Expand-Mobile-putfile-RPC.md
+-- User story:TBD
+-- Use case:TBD
+--
+-- Requirement summary:
+-- TBD
+--
+-- Description:
+-- In case: PT is updated without requestSubType for application App2 and App2 starts regisration
+-- SDL does: not send requestSubType in OnAppRegistered during registration
```

Comment:
@GetmanetsIrina In case if in PTU there was no 'requestSubType' parameter, values from 'default' section are used
In this case if default section contains empty array, OnAppRegistered will be sent with empty array for 'requestSubType' parameter